reference/providers/kubernetes-v2/index.md

```diff
@@ -120,6 +120,16 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
   When set to `true`, tells Spinnaker to ignore this resource.
   The resource is not cached and does not show up in the Spinnaker UI.
 
+## Strategy
+
+* `strategy.spinnaker.io/versioned`
+
+  When set to `true` or `false`, this overrides the resource's default
+  "version" behavior described in the [resource management
+  policies](#resource-management-policies). This can be used to force a
+  ConfigMap or Secret to be deployed without appending a new version when the
+  contents change, for example.
```

Comment:
@lwander do you want to also add a sentence to the resource management policies section mentioning this?

Reply:
Good idea!

Reply:
Done

Reply:
:+1: